src/resources/UsageAnalytics/Read/Exports/ExportsInterfaces.ts

```diff
@@ -87,10 +87,88 @@ export interface GenerateExportParams extends EstimateExportParams {
     useDisplayNames?: boolean;
 }
 
+interface BaseGenerateExportWithBodyParams {
+    /**
+     * The beginning date of the date range. ISO8601 format 'YYYY-MM-DDThh:mm:ss.sssZ'
+     */
+    from: string;
+    /**
+     * The end date of the date range. ISO8601 format 'YYYY-MM-DDThh:mm:ss.sssZ'
+     */
+    to: string;
+    /**
+     * A file name for the export. Can only contain US-ASCII characters. If none is supplied it will be set to the export's id.
+     */
+    filename?: string;
+    /**
+     * Optional description for the export.
+     */
+    description?: string;
+    /**
+     * The dimensions that will be exported. If not provided, all dimensions will be exported.
+     */
+    dimensions?: string[];
+    /**
+     * The format of the generated CSV files. If not provided, it will default to Excel.
+     */
+    exportFormat?: CSVFileFormat;
+    /**
+     * Whether to use the display names in the export's header. If false, the api names will be used. If not provided, it defaults to false.
+     */
+    useDisplayNames?: boolean;
+}
+
+export interface GenerateExportWithBodyParams extends BaseGenerateExportWithBodyParams {
+    /**
+     * The filter that will be applied to the events common dimensions (shared by all types of events).
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    commonFilters?: string[];
+    /**
+     * The filter that will be applied to the click and search events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    searchesFilters?: string[];
+    /**
+     * The filter that will be applied to the custom events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    customEventsFilters?: string[];
+    /**
+     * The tables that will be exported. If not provided, all tables will be exported.
+     */
+    tables?: ExportTablesType[];
+}
+
+export interface GenerateVisitExportWithBodyParams extends BaseGenerateExportWithBodyParams {
+    /**
+     * The filter that will be applied to the events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    inclusionFilters?: string[];
+    /**
+     * Each specified filter is inverted in order to hide events.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    hideEventFilters?: string[];
```

Comment:
I'm not sure I understand the point of this parameter if we already have inclusion and exclusion filters.

Also, what will happen if both inclusion and exclusion filters are provided? I presume they will be merged, and exclusion will "win" if mutually exclusive filters are sent, but I think the behavior should be documented.

This behavior should be documented.

Reply:
> I'm not sure I understand the point of this parameter if we already have inclusion and exclusion filters.

You can see their intended usage in the public docs here- https://docs.coveo.com/en/1964/analyze-usage-data/review-user-visits-with-the-visit-browser#hide-events

Yes, exclusion filter will have precedence, if you specify the same filter as both an inclusion and exclusion filter- I'll add this to the comment.

Reply:
I read the docs and I can see how this could be of use if the report is rendered in the admin UI, but I still don't understand what's the difference between hiding and excluding events in a CSV data export. I'm quite confused by this parameter 😅